R/tbl_propdiff.R

```diff
@@ -0,0 +1,365 @@
+#' Calculating unadjusted and adjusted differences in rates
+#'
+#' This function calculates the unadjusted or adjusted difference in rates with confidence interval.
+#'
+#' @param data A data frame
+#' @param outcome A vector of outcome variable names. Outcome variables can be numeric, character or factor, but must have two and only two non-missing levels
+#' @param predictor The name of the predictor (stratifying) variable. This can be numeric, character or factor, but must have two and only two non-missing levels
```

Comment:
We should update to accept a vector of predictors.  But this can be put on hold until i update gtsummary `tbl_stack()` to work with all gtsummary-like objects.

Reply:
we discussed and decided not to do this, will take a vector of outcomes but only one predictor